src/components/pages/Help/HelpContainer.js

```diff
@@ -1,41 +1,18 @@
-import React, { useState, useEffect, useMemo } from 'react';
-import { useOktaAuth } from '@okta/okta-react';
+import React, { useState } from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
 
 import Help from './Help';
 
 function HelpContainer({ LoadingComponent }) {
-  const { authState, authService } = useOktaAuth();
+  const { user, isAuthenticated } = useAuth0();
   const [userInfo, setUserInfo] = useState(null);
```

Comment:
Hey Seth! I just noticed here on line 8 that the useState is set to null. In the other files we have it initialized with "user".

Reply:
Good catch! I thought I had them all, but this one was missed. I'll push these changes now.

Reply:
Thanks! It does make me want to go back and look at all of mine now. I certainly could have missed the same thing.